docs/client/AccountConsentApi.md

```diff
@@ -0,0 +1,66 @@
+# \AccountConsentApi
+
+All URIs are relative to *http://api.pureport.com*
+
+Method | HTTP request | Description
+------------- | ------------- | -------------
+[**Consent**](AccountConsentApi.md#Consent) | **Post** /accounts/{accountId}/consent | Consent to Master Service Agreement and Acceptable Use Policy for this account
+[**HasConsent**](AccountConsentApi.md#HasConsent) | **Get** /accounts/{accountId}/consent | Account consent to Master Service Agreement and Acceptable Use Policy
+
+
+# **Consent**
+> Consent(ctx, accountId)
+Consent to Master Service Agreement and Acceptable Use Policy for this account
+
+
+
+### Required Parameters
+
+Name | Type | Description  | Notes
+------------- | ------------- | ------------- | -------------
+ **ctx** | **context.Context** | context for authentication, logging, cancellation, deadlines, tracing, etc.
+  **accountId** | **string**|  | 
+
+### Return type
+
+ (empty response body)
+
+### Authorization
+
+No authorization required
```

Comment:
All of these methods that say "No Authorization required", does it mean no auth required as parameters in this call or not auth overall, because I'm pretty sure most of these need some auth or they won't work. Maybe not this call, but others.

Reply:
This is b/c the authentication method for these end-points are being added to the Swagger-API.
I'll start adding defects for implementing these so we don't lose track.

Reply:
https://pureport.atlassian.net/browse/PE-1042